docs/en/ros2/px4_ros2_control_interface.md

```diff
@@ -400,6 +401,96 @@ _goto_setpoint->update(
   max_heading_rate_rad_s);
 ```
 
+#### Fixed-Wing Lateral and Longitudinal Setpoint (FwLateralLongitudinalSetpoint)
+
+::: info
+This setpoint type is only supported for fixed-wing vehicles.
+:::
+
+[//]: # (TODO: ADD LINKS TO CODE BLOBS AFTER MERGING)
+Use the px4_ros2::FwLateralLongitudinalSetpointType to directly control the lateral and longitudinal dynamics of a fixed-wing vehicle. This setpoint is streamed to the PX4 FwLateralLongitudinalControl module, which decouples lateral and longitudinal inputs while ensuring that vehicle limits are respected.
+
+For a clear explanation of the minimum required set of compatible inputs, refer to the message definitions.
+(TODO: ADD LINK HERE)
+
+##### Basic Usage
+
+In the simplest case, you can provide only a course and altitude setpoint:
+```cpp
+const float altitude_msl = 500.F;
+const float course = 0.F; // due North
+_fw_lateral_longitudinal_setpoint->update(altitude_msl, course);
+```
+PX4 will compute:
+
+- Lateral control output: required airspeed, airspeed direction (heading), lateral acceleration, and roll angle
+
+- Longitudinal control output: height rate, pitch angle, and throttle settings
+
+To additionally control the height rate, equivalent airspeed, or lateral acceleration, you can specify them as the third, fourth, and fifth arguments, respectively:
+
+```cpp
+const float altitude_msl = 500.F;
+const float course = 0.F; // due North
+const float equivalent_aspd = 15.F; // m/s
+const float lateral_acceleration = 2.F; // FRD, used as feedforward
+
+_fw_lateral_longitudinal_setpoint->update(altitude_msl,
+  course,
+  std::nullopt,
+  equivalent_aspd,
+  lateral_acceleration);
+```
+
+The height rate, equivalent airspeed, and lateral acceleration arguments are defined as `std::optional<float>`, so you can omit any of them by passing `std::nullopt`. In the example above, the height rate is left unset.
+
+::: tip
+If multiple lateral inputs are set, the lower level inputs are used as feedforward. If both altitude and height rate are set, *altitude is not controlled*.
```

Comment:
> If multiple lateral inputs are set, the lower level inputs are used as feedforward

For which combinations does that work? Maybe worth to specifically point out the possible combinations? Unless it's too many options?